### ISSUE TITLE
Add headings to appointments

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/calendar#4126]
### Is your feature request related to a problem? Please describe.

From https://github.com/nextcloud/calendar/issues/3540

> Every appointment should have a heading so it’s possible to directly navigate to it, including the time and title

### Describe the solution you'd like

N/a

### Additional context

@jancborchardt @nimishavijay would it make sense to have a rough mockup of where this should be added and how it should look?
